### PR TITLE
Meet the team

### LINKED ISSUE #231: Meet the Team Tooltip Removal + Linkedins
Based on the [design](https://www.figma.com/file/ONkXtJcw5iJL5Az7Mmi64S/Main-Site-Wireframes?type=design&node-id=300-660&mode=design&t=yEGSrBITKIzUPJCw-0), we will be removing the tooltip functionality and replacing it with LinkedIn links below each bubble!

[Linkedins](https://docs.google.com/spreadsheets/d/1S0Uak2t9wslm-D3Z02vvQ2dNx3aUGUYAhpnHFOHQzTg/edit#gid=0)

Image

**Acceptance Criteria**

- [ ] Remove all tooltip functionally 
- [ ] Add LinkedIn links under headshot bubbles to reflect [design](https://www.figma.com/file/ONkXtJcw5iJL5Az7Mmi64S/Main-Site-Wireframes?type=design&node-id=300-660&mode=design&t=yEGSrBITKIzUPJCw-0)

@mike10911 @jmsargent10

### PR DESCRIPTION
**Pull request by Aaryan1203**
Added LinkedIn's and removed tooltip
Fixes #231

Changelist:

- Removed Tooltip throughout all files
- Changed the type 'person' to include LinkedIn's and all of the information from tooltip
- Added LinkedIn under each profile picture

Notes:

- Mobile View: The pictures are no longer in groups of two but instead in one single column which needs to be fixed

Screenshots & Screencasts:
Desktop View:
image

Mobile View:
image

This is what the old mobile view looked like with the pairs of two:
image

### CODE REVIEW

File: shared-ui/components/meet-the-team/team-column/desktop-team-column/DesktopTeamColumn.styles.ts [hunk 1]
```diff
@@ -30,4 +30,38 @@ const StyledHeadshot = styled.img`
   }
 `;
 
-export { StyledTeamColumn, StyledHeadshot };
+const StyledLinkedIn = styled.a`
```
Comment (mike10911) [inline shared-ui/components/meet-the-team/team-column/desktop-team-column/DesktopTeamColumn.styles.ts:33]: This needs to match the font on the designs.

**Review by mike10911 — CHANGES_REQUESTED** [review]
Slay just one comment!

**Review by jmsargent10 — CHANGES_REQUESTED** [review]
Approved with small change! I think some of the linkedin links are incorrect / outdated including both yours and mine so they don't actually direct to our linkedins! Just make sure all the links are functioning as intended, but other than that LGTM!!

My linkedin is: [https://www.linkedin.com/in/john--sargent/](url)

**Review by mike10911 — APPROVED** [review]
LGTM. We can look into fixing the staggering in the future

**Review by mike10911 — CHANGES_REQUESTED** [review]
merge into `under-the-sea` branch

**Pull request closed without merging**